public/scss/components/CartDetails.module.scss

```
@@ -131,41 +130,175 @@
   }
 }
 
+.itemPrice
+{
+  @include grid-start-(3,2);
+  align-self: center;
+}
+
 .itemSalePrice
 {
   @include SmallSemiBold;
-  color: $color_error
+  color: $color_error;
+}
+
+.itemQty
+{
+  @include grid-start-(2, 2);
 }
 
 .qtyBox
 {
-  @include SmallSmall;
-  display: flex;
-  align-items: center;
+  position: relative;
 
-  input
+  /* Chrome, Safari, Edge, Opera */
+  input::-webkit-outer-spin-button,
+  input::-webkit-inner-spin-button {
+    -webkit-appearance: none;
+    margin: 0;
+  }
+
+  /* Firefox */
+  input[type=number] {
+    -moz-appearance: textfield;
+  }
```

```suggestion
@include removeArrowsFromInput;
```

ini ada mixin-nya ngab, ada di `public/scss/utils/_mixins.scss` line `147`